client/src/i18n/en/stock.json

```diff
@@ -101,6 +101,16 @@
     "MONTH"           : "Stock Months",
     "MONTH_OF_STOCK"  : "Stock Months - Months until Stock Out",
     "MONTHLY_CONSUM"  : "Average Monthly Consumption",
+    "MONTHLY_CONSUMPTION" : {
+      "ALGO_1"          : "Algorithm 1",
+      "ALGO_1_COMMENT"  : "The average monthly consumption is obtained by dividing the quantity consumed during the period defined by the number of days of stock for said period, and by multiplying the result obtained by 30.5",
```

Comment:
Since you mentioned `monthly consumption` and `30.5` which are related to month, is it possible to change `period defined` by just `month`?
Ex. The average monthly consumption is obtained by dividing the quantity consumed during the month by the number of days of stock of the month, and by multiplying the result obtained by 30.5

Reply:
I had agreed with @mbayopanda, for the calculation of the CMM we do not use only one me but rather the period defined in the stock Setting 
```
month_average_consumption 
````

Reply:
We should probably come up with a better name than "algo 1", "algo 2", etc.  Also, I'm not sure we should support anything other than algo 1 and 4.